### PR TITLE
Document connecting to ssh Tor Hidden Service

### PR DESCRIPTION
**Pull request by nixbitcoin**
This is a first step in more extensive documentation of this project. Eventually the documentation in the README should be split up into an INSTALL.md, USEAGE.md, and FAQ.md. I think a good time to do this would be after we move user configuration functionality from modules/nix-bitcoin.nix to configuration.nix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +164,46 @@ FAQ
...
+	services.openssh.enable = true;
+	users.users.root = {
+	  openssh.authorizedKeys.keys = [ "[contents of ~/.ssh/id_ed25519.pub]" ];
+	};
+	```
+
+	4. Connect to your nix-bitcoin node's ssh Tor Hidden Service, forwarding a local port to the nix-bitcoin node's ssh server
+
+	```
+	ssh -i ~/.ssh/id_ed25519 -L [random port of your choosing]:localhost:22 root@[your SSHD_ONION]
```
Comment (jonasnick) [inline README.md]: the ssh key you created above is not necessarily in ~/.ssh/id_ed25519

File: README.md [hunk 2]
```diff
@@ -164,6 +164,46 @@ FAQ
     * **A:** Check your clightning logs with `journalctl -eu clightning`. Do you see something like `bitcoin-cli getblock ... false` failed? Are you using pruned mode? That means that clightning hasn't seen all the blocks it needs to and it can't get that block because your node is pruned. If you're just setting up a new node you can `systemctl stop clightning` and wipe your `/var/lib/clightning` directory. Otherwise you need to reindex the Bitcoin node.
 * **Q:** My disk space is getting low due to nix.
     * **A:** run `nix-collect-garbage -d`
+* **Q:** How do I connect to my nix-bitcoin node through the ssh Tor Hidden Service?
+    * **A:** 
+
+	1. Run `nodeinfo` and note the `SSHD_ONION`
```
Comment (jonasnick) [inline README.md]: ... on your nix-bitcoin node perhaps?

File: README.md (hunk 1 above)
Comment (nixbitcoin) [reply README.md]: Fixed in be3d4ed

File: README.md (hunk 2 above)
Comment (nixbitcoin) [reply README.md]: Fixed in be3d4ed

Comment (nixbitcoin) [reply README.md]: Also added a code snippet to make "run nodeinfo" more clear.

**Pull request merged**